bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -86,18 +86,25 @@ spec:
           fi
 
           # And finally start the ovs-vswitchd now the DB is prepped
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --no-ovsdb-server --system-id=random
+          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --no-ovsdb-server --system-id=random --no-monitor
 
+          /usr/share/openvswitch/scripts/ovs-ctl status > /dev/null &&
+          /usr/bin/ovs-appctl -T 5 ofproto/list > /dev/null &&
+          /usr/bin/ovs-vsctl -t 5 --retry show > /dev/null
```

Comment:
If the goal of this is "make sure that the database has started" then there is probably a better way... At least, it seems like we should only need one of these commands?

Reply:
The goal is to make sure database has started and the ovs-vswitchd is up so it can take ovs-ofctl commands.

Reply:
I would think probably `ovs-ctl status` alone should be sufficient for that? You definitely shouldn't need three different commands. I forget exactly what they're all doing in the readiness check, but that may not apply here.

Reply:
Yes, `ovs-ctl status` will indicate if the daemons are running. The other two commands are useful to check for socket readiness for ovs-vswitchd and ovsdb-server respectively. Given that restore flows only uses `ovs-ofctl`, I can get rid of the `ovs-vsctl` check. I would prefer to keep the `ovs-appctl` check since we have seen a lot of socket errors at start up in the logs before.